src/openloco/windows/build_vehicle.cpp

```diff
@@ -1046,7 +1044,12 @@ namespace openloco::ui::build_vehicle
                     {
                         auto cargoObj = objectmgr::get<cargo_object>(cargoType);
                         FormatArguments args{};
-                        args.push(vehicleObj->max_primary_cargo == 1 ? cargoObj->unit_name_singular : cargoObj->unit_name_plural);
+                        auto cargoUnitName = cargoObj->unit_name_plural;
+                        if (vehicleObj->max_primary_cargo == 1)
+                        {
+                            cargoUnitName = cargoObj->unit_name_singular;
+                        }
```

Comment:
Oh, you didn't have to get rid of the ternary, really. I was just thinking something among the lines of:
```suggestion
                        auto cargoUnitName = vehicleObj->max_primary_cargo == 1 ? cargoObj->unit_name_singular : cargoObj->unit_name_plural;
```

But this construction is fine, too, of course.